### DISCUSSION TITLE
翻訳ガイドをリライト版ついて

### DISCUSSION THREAD

**Discussion by gurezo** [mozilla-japan/translation#680] [Ideas]
### 目的
下記ポストの認識合わせを @debiru さんと皆でしたい。
slack だと[ログ](https://mozillajp.slack.com/archives/C5N6SFQMP/p1674900117985309)が消えてしまうので、こちらに転記して記録を残したい

cc @hmatrjp @potappo 

2023.01.18 19:08 @debiru posted
```
月例ミートアップの翻訳ガイドをリライトしてみました。
現行版： https://mozilla-japan.github.io/mdn-translation-guide/
リライト版： https://mdn.debiru.net/
リライト版について何かご意見等があれば発言いただけると助かります。
もし、問題がなさそうなら翻訳ガイドの差し替えがで
```

2023.01.18 22:22 @potappo posted
```
おおむね良さそうと思います。ざっと読んで気になった点を。
タイムスケジュールのクロ―ズは18:00ですね（イベント案内上の終了時間は18:00になっています）。
「リンク集」のページは（サイドメニューなしのスマホで読んでいたからしれないですが）突然出てきて違和感があったので、「便利なリンク集」のままの方がリンク集の目的が伝わって良いかもしれません。「便利な」以外に適当な言葉があれば、それでもかまいません。
```

2023.01.18 19:08 @debiru posted
```
確認ありがとうございます。ご指摘の箇所は修正いたします…！リンク集についても考えてみますね。
```

2023.01.18 19:08 @debiru posted
```
タイムスケジュールを修正しました。links は misc へ移動しました。
```

### 確認したいこと
リライト版： https://mdn.debiru.net/

- 上記のリライト版の翻訳ガイドは、[ミートアップ資料の見直し #2 ](https://github.com/mozilla-japan/mdn-translation-guide/issues/2) について、 @debiru さんが考えた構成提案を自分のドメインにデプロイして、レビュー・評価して欲しいという理解であっていますか？
- 違う意図があったのでしょうか？

### 気にしていること
- 翻訳ガイドが２重管理になってしまわないか、メンテナンスが２倍になってしまわないか
  - 現行版： https://mozilla-japan.github.io/mdn-translation-guide/
  - リライト版： https://mdn.debiru.net/

**Comment by gurezo** [comment]
> 上記のリライト版の翻訳ガイドは、[ミートアップ資料の見直し #2 ](https://github.com/mozilla-japan/mdn-translation-guide/issues/2) について、 @debiru さんが考えた構成提案を自分のドメインにデプロイして、レビュー・評価して欲しいという理解であっていますか？

個人的には上記意図であれば、フォークしたリポジトリで、Githubページを公開した上で
PR などを出して頂いた方が良いかな？と思いました。

e.g. https://gurezo.github.io/mdn-translation-guide/

**Reply by debiru** [reply]
> 個人的には上記意図であれば、フォークしたリポジトリで、Githubページを公開した上で
> PR などを出して頂いた方が良いかな？と思いました。

仰る通りだと思います。フォークすればよかったですね。
HonKit を使ったこともなかったので、HonKit のデプロイ手順を試す意味も含めて自分のリポジトリで作成してしまいました。

> 翻訳ガイドが２重管理になってしまわないか、メンテナンスが２倍になってしまわないか

あくまでメインの管理場所は https://mozilla-japan.github.io/mdn-translation-guide/ であるという認識で、私のリポジトリはサンドボックスだと思っていただいて大丈夫です。https://mdn.debiru.net/ はお試しで作ってみただけの場所でした。

私の方で PR が作れるようにフォークしてリポジトリを作り直して、それをレビューしてもらった方がよさそうですね。失礼しました。

**Reply by gurezo** [reply]
お返事ありがとうございます。

> 私の方で PR が作れるようにフォークしてリポジトリを作り直して、それをレビューしてもらった方がよさそうですね。失礼しました。

いえいえ、意図が理解できて良かったです。
PRをお待ちしてます:bow:

**Comment by potappo** [comment]
PR されてマージされているようなので、このディスカッションはクローズします。
